docs/sectoral_coverage/power-sector.md

```diff
@@ -1,3 +1,26 @@
 ## Power Sector
 
-In progress.
\ No newline at end of file
+The electricity or power generation sector represents many different types of technologies ranging from fossil-based coal power plants to renewable technologies, i.e. PV and wind power plants.TIAM model focusses explicitly on renewable power plants- Solar PV, Biomass, Hydropower, Nuclear, Onshore and Off-shore wind power plants. The power sector Base-Year template is employed for calibration of base year electricity generation. The complete electricity production is fed into the grid/transmission network for use in energy demand sectors. Off-grid power production (micro-generation technologies) is not yet modelled in TIAM.
+In terms of economic evaluation, TIAM considers explicitly CAPEX, OPEX, efficiency and lifetime of the power plants for electricity supply. The model considers unlimited commodities/resources for renewable plants. Unlike coal or oil which needs expenses for production; renewable sources- solar radiation or wind energy are supplied at no costs. Hence, no additional expenses for fuel are considered.  As explained in time resolution section, TIAM model has ten different periods. The table below shows CAPEX costs for four different time periods. It is to explain the trend in declining CAPEX as the technologies get more mature and will be less expensive to invest in the future. 
```

Comment:
https://etsap-tiam.readthedocs.io/en/latest/restrictions/potentials-renewable.html

Reply:
@LippkauFelix should anything happen to this URL?

Reply:
Ah sorry, something happend to the text that I wrote. I would propose to add a reference to: 

> The model considers unlimited commodities/resources for renewable plants

Actually the renewables are restricted by using a capacity constraint, which is described using the linked page in the documentation.

Reply:
Ok, thanks! Can add a cross reference there.

Reply:
@LippkauFelix please check. Is it how you meant?

Reply:
Can't find it on the pre-rendered page.

Reply:
Seems those changes were overwritten by an update... I've put them back now.